index.js

```diff
@@ -15,41 +23,13 @@ app.set("view engine", "handlebars");
 app.use(express.static(__dirname + "/static"));
 
 app.use("/", homeRouter);
-
-app.get("/learn", (req, res, next) => {
-  res.render("learn");
-});
-
-app.get("/courses", (req, res, next) => {
-  res.render("courses");
-});
-
-app.get("/workshops", (req, res, next) => {
-  res.render("workshops");
-});
-
-app.get("/about", (req, res, next) => {
-  fs.readFile("data/team.json", (err, data) => {
-    if (err) throw err;
-    res.render("about", JSON.parse(data));
-  });
-});
-
-app.get("/contact", (req, res, next) => {
-  res.render("contact");
-});
-
-app.get("/support", (req, res, next) => {
-  res.render("support");
-});
-
-app.get("/blog", (req, res, next) => {
-  res.render("blog");
-});
-
-//The 404 Route (ALWAYS Keep this as the last route)
-app.get("*", function(req, res) {
-  res.render("error");
-});
+app.use("/", aboutRouter);
+app.use("/", blogRouter);
+app.use("/", contactRouter);
+app.use("/", coursesRouter);
+app.use("/", learnRouter);
+app.use("/", supportRouter);
+app.use("/", workshopsRouter);
+app.use("/", errorRouter);
```

Comment:
Sorry, all of these are wrong :)
You need to use paths here, and in the router files, use **/**

So, like the following
```
app.use("/learn", learnRouter);
```

And in learnRouter:
```
router.get("/", (req, res, next) => {
  res.render("learn");
});
```

Reply:
according to my node.js course this is the correct way:
`app.use(learnRouter);`
And in learnRouter:
`router.get("/learn", (req, res) => {
  res.render("learn");
});`

Do you agree?